extensions/wikia/ContentReview/ContentReview.i18n.php

```diff
@@ -49,6 +49,10 @@
 Names should not contain the MediaWiki namespace prefix. Write each script on a new line. See [[Help:Including additional CSS and JS]] for more information.
 ----
 ',
+	'content-review-user-badges-description' => 'Here, you can easily add custom badges to users. Just separate user name and rights by colon. If there is more rights list them using comma,
```

Comment:
* `by colon` -> `by a colon`
* `If there is more rights list them using comma` -> `If there are more rights, list them using comma`

I'd also consider consistently saying "badge" rather than "rights" since these aren't actually rights.